pkg/security/secl/model/model.go

```diff
@@ -193,6 +211,13 @@ type Event struct {
 	Exit     ExitEvent     `field:"exit" event:"exit"`     // [7.38] [Process] A process was terminated
 	Syscalls SyscallsEvent `field:"-"`
 
+	// anomaly detection related events
+	AnomalyDetectionSyscallEvent AnomalyDetectionSyscallEvent `field:"-"`
+	AnomalyDetectionOpenEvent    AnomalyDetectionOpenEvent    `field:"-"`
+	AnomalyDetectionProcessEvent AnomalyDetectionProcessEvent `field:"-"`
+	AnomalyDetectionBindEvent    AnomalyDetectionBindEvent    `field:"-"`
+	AnomalyDetectionDNSEvent     AnomalyDetectionDNSEvent     `field:"-"`
```

Comment:
I might have missed something but I don't think we need these 🤔 (the anomaly detection syscall one was an exception because we don't have an OOTB event that specifically accounts for 1 syscall and not a list of syscalls)

Reply:
Yeah I was wondering too, ok let's remove them, thanks